server/src/style/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Shouldn't we have a copyright header here?

Reply:
Fixed